### PR TITLE
Update release notes for v2.0.1

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

**Comment by bitcartel** [comment]
From @ebfull "ACK from my phone. can’t do it on github atm"

**Comment by bitcartel** [comment]
@zkbot r+

File: doc/release-notes.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ release-notes at release time)
 Notable changes
 ===============
 
+Enabled Sapling features for mainnet
+------------------------------------
+This release adds significant support for Sapling to the wallet and RPC interface. Sapling will activate at block 419200, which is expected to be mined on the 28th of October 2018. Users running v2.0.0 nodes (which are consensus-compatible with Sapling) will follow the network upgrade, but must upgrade to v2.0.1 in order to send Sapling shielded transactions.
```
Comment (daira) [inline doc/release-notes.md]: "send or receive"

File: doc/release-notes.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ release-notes at release time)
...
+
+Minimum Difficulty Blocks allowed on testnet
+--------------------------------------------
+Sapling activated on testnet at block 280000. Users running v2.0.1 nodes no longer need to specify `-experimentalfeatures` and `-developersapling` to use Sapling functionality on testnet. Users running v2.0.0 nodes should upgrade to v2.0.1 which introduces a consensus rule change to allow minimum difficulty blocks to be mined from block 299188, thereby splitting the chain.
```
Comment (daira) [inline doc/release-notes.md]: The subheading doesn't reflect all of the content of this paragraph.

File: doc/release-notes.md [hunk 3]
```diff
@@ -17,3 +27,9 @@ Regular backups are still necessary, however, in order to ensure that
 transparent and Sprout addresses are not lost.
 
 [Pull request](https://github.com/zcash/zcash/pull/3492), [ZIP 32](https://github.com/zcash/zips/blob/master/zip-0032.mediawiki)
+
+Fix Signing Raw Transactions with Unsynced Offline Nodes
+--------------------------------------------------------
+With v2.0.0, in `signrawtransaction` the consensus branch ID (which is used to construct the transaction) was estimated using the chain height. With v2.0.1 this has been improved by also considering the `APPROX_RELEASE_HEIGHT` of the release, and a new parameter to allow the caller to manually override the consensus branch ID that zcashd will use.
```
Comment (daira) [inline doc/release-notes.md]: Is it true that 2.0.0 uses estimated chain height? I don't remember that.

Comment (bitcartel) [reply doc/release-notes.md]: Yes, for never synced nodes, this meant 0. https://github.com/zcash/zcash/pull/3520/files#diff-01aa7d1d32f1b9e5a836c9c411978918L877

**Pull request merged**